ble_controller/include/ble_controller_hci_vs.h

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file ble_controller_hci_vs.h
+ *
+ * @defgroup ble_controller_hci_vs BLE Controller Vendor Specific HCI Interface
+ * @ingroup ble_controller
+ *
+ * The BLE Controller VS HCI APIs define additional functionality provided by
+ * the BLE controller library.
+ * All APIs in this header file are expected to be called from the
+ * same execution priority as @ref ble_controller_low_prio_tasks_process.
+ * Not doing so will lead to undefined behavior.
+ * @{
+ */
+
+#ifndef BLE_CONTROLLER_HCI_VS_H__
+#define BLE_CONTROLLER_HCI_VS_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
+#include "cmsis_compiler.h"
```

Comment:
Where is this file?
Is it right to include it with "" instead of <>?

Reply:
In NCS includes should use " if and only if the file path is relative to the file doing the including.

Reply:
Will switch to <>

Reply:
Valid question, I am sorry for missing it.

Our rationale is as follows:
- When using the controller along with zephyr you will get the header in 
  `zephyr/ext/hal/cmsis/Include`
- When using the controller standalone, you need the mdk, 
  and the mdk has a dependency on cmsis.
  For example, in `nrf52840.h` one can find `#include "core_cm4.h"`

Reply:
Ok. I guess that is good enough.